.github/workflows/python-package.yml

```
@@ -29,7 +29,7 @@ jobs:
     - name: Install dependencies
       run: |
         pip install --upgrade pip
-        pip install -e .[test]
+        pip install -e .[test,experimental]
```

```suggestion
        pip install -e .[test]
```

The experimental grouping was deleted, if I'm not mistaken

---

That looks correct